### ISSUE TITLE
Problems on create place from reference resource by URI

### ISSUE THREAD

**Issue by skurzinz** [acdh-oeaw/apis-core#178]
Curator cannot create some places from reference resources:

- Predil https://www.wikidata.org/wiki/Q1754407 | http://d-nb.info/gnd/4437197-4 | http://sws.geonames.org/3192559/about.rdf  
remark: the gnd URI is not parsed, attempt to retrieve data only works if I remove the "-4" from the uri. 
- similarly, Stein an der Donau  http://sws.geonames.org/2764553/about.rdf gives back "The results could not be loaded"

**Comment by sennierer** [comment]
did that happen on the production instance?

**Comment by skurzinz** [comment]
yes, production instance it was. can you reproduce it or do you need screenshots or the like?

**Comment by sennierer** [comment]
I am having trouble to reproduce that:
* there are no settings for parsing wikidata yet, so if you tried to parse the wikidata rdf that wont work
* the GND parses fine, the problem is only that is not in our stanbol index (as it is not a city) and therefore you dont get a hit from the autocomplete
* same goes for the geonames link, parses just fine, but is a "Pass" and therefore not in our stanbol index
* Same for Stein
